### PR TITLE
UP-3758 Implements Delete Permission UI stubs

### PR DESCRIPTION
**Pull request by vertein**
The delete permission ui works now.

Before: 
deletepermissionbad

After:
Page Exists:
deletepageexists

Cancel and Breadcrumbs work:
breadcrumbsandcancel

Remove Button works:
removebuttondoesntcrash

### CODE REVIEW

File: uportal-war/src/main/webapp/WEB-INF/flows/permissions-administration/deletePermission.jsp [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        
+</div> <!-- end: portlet -->
+
+<script type="text/javascript">
+up.jQuery(function() {
+    var $ = up.jQuery;
+    
+    $(document).ready(function(){
+        $('#${n}deletePermissionButton').click(function(e){
+            var principals = [<c:forEach items="${ principals }" var="principal" varStatus="status">'${ principal }'${ status.last ? '' : ', '}</c:forEach>];
```
Comment (jameswennmacher) [inline uportal-war/src/main/webapp/WEB-INF/flows/permissions-administration/deletePermission.jsp]: I'm not clear on this.  Is it used somewhere?

**Comment by jameswennmacher** [comment]
Looks good!

File: uportal-war/src/main/webapp/WEB-INF/flows/permissions-administration/deletePermission.jsp (hunk 1 above)
Comment (vertein) [inline uportal-war/src/main/webapp/WEB-INF/flows/permissions-administration/deletePermission.jsp]: Hey @jameswennmacher , you're correct.  Principals was used at one point but forgot to take it out.  Good catch!  Will respond when the pull request is updated.

Comment (vertein) [inline uportal-war/src/main/webapp/WEB-INF/flows/permissions-administration/deletePermission.jsp]: Updated

**Comment by timlevett** [comment]
:+1:

**Pull request merged**